src/main/java/mil/dds/anet/AnetCheckCommand.java

```diff
@@ -16,7 +16,7 @@ public AnetCheckCommand(Application<AnetConfiguration> application) {
   protected void run(Bootstrap<AnetConfiguration> bootstrap, Namespace namespace,
       AnetConfiguration configuration) throws Exception {
     // Check the dictionary
-    AnetApplication.getDictionary(configuration);
+    configuration.checkDictionary();
```

Comment:
I suspect this will fail as `loadDictionary` has not happened yet

Reply:
After updating the way I've loaded dictionary I've "checked bin/anet check anet.yml" command . It's ok now.